plural/helm/console/Chart.yaml

```diff
@@ -2,17 +2,17 @@ apiVersion: v2
 name: console
 description: A chart for plural console
 appVersion: 0.10.40
-version: 0.10.67
+version: 0.10.68
```

Comment:
can we not version bump this until we do the release?  Will make it easier to test

Reply:
Ye, I will revert